vignettes/approved.Rmd

```diff
@@ -146,7 +146,10 @@ close(fileConn)
 ```
 
 ```{r results='hide'}
-axecute(file.path(dir,"mpg.R"), remove_log_object = FALSE)
+fp <- file.path(dir,"mpg.R")
+log_config(fp)
+logrx:::run_safely_loudly(fp)
+log_write(fp, remove_log_object = FALSE)
```

Comment:
I feel like this is confusing to users, since they would not execute a program this way.  Maybe we still show the axecute code but in a separate code block that is hidden, we do this so we can just get the approved log chunk to write here.

Reply:
Agree with this suggestion after considering the main purpose of the vignette. This pattern isn't useful to surface.